meta-oe/recipes-devtools/uftrace/uftrace_0.9.4.bb

```diff
@@ -10,6 +10,7 @@ DEPENDS_append_libc-musl = " argp-standalone"
 
 inherit autotools
 
+PV .= "+git${SRCPV}"
 SRCREV = "d648bbffedef529220896283fb59e35531c13804"
 SRC_URI = "git://github.com/namhyung/${BPN} \
            "
```

Comment:
You should remove the same from PR added in:
https://git.openembedded.org/meta-openembedded/commit/?id=f60d1303bb2df6702d6dc972aaeecd8d75e73cac

Reply:
@AdrianBunk Why did you use PR instead of PV? Was it just a typo?

Reply:
Thanks. I didn't see that. Is that okay if I just replace `PR .= "+git${SRCPV}"` to `PV .= "+git${SRCPV}"`?

Reply:
yes

Reply:
> @AdrianBunk Why did you use PR instead of PV? Was it just a typo?

Yes it was, sorry for that.